packages/terra-form-select/CHANGELOG.md

```diff
@@ -5,7 +5,10 @@
 ## 6.51.0 - (November 21, 2023)
 
 * Added
-  * Added 'aria-invalid' attribute which will be set to true for error input fields and false when resolving errors. 
+  * Added 'aria-invalid' attribute which will be set to true for error input fields and false when resolving errors.
+
+* Fixed
+  * Fixed ScreenReader reading undefined selected and left and parent extra information in `terra-form-select-combobox`.
```

Comment:
Move this entry to unreleased section 
```suggestion
* Fixed
   * Fixed screen reader response for `terra-form-select-combobox`.
```

Reply:
Updated - https://github.com/cerner/terra-core/pull/3985/commits/3f70c696c7305bb72b122d9df43479b2de067edc